test/models/research/experiment_solution_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
Can we remove these empty tests?

Reply:
Sure. Want to do it when you cherry-pick?